modules/cnv-example-bond-nncp.adoc

```diff
@@ -8,6 +8,14 @@
 Create a bond interface on nodes in the cluster by applying a `NodeNetworkConfigurationPolicy` manifest
 to the cluster. 
 
+[NOTE]
+====
+{CNVProductNameStart} 2.3 only supports the following bond modes: +
+* mode=1 active-backup +
```

Comment:
The asterisk is not rendered as a list item in the doc. Maybe we should drop them. I'm not an UX person though.

Reply:
No, you're right. It was missing a linebreak. I either need glasses or to change my monitor resolution